client/pages/game.jsx

```diff
@@ -11,20 +12,40 @@ export default class Game extends React.Component {
     this.state = {
       board: new Board(),
       gamestate: new GameState(),
-      meta: null
+      meta: null,
+      side: null,
+      socket: io()
     };
     this.cancelGame = this.cancelGame.bind(this);
   }
 
   componentDidMount() {
-    const gameId = parseRoute(window.location.hash).params.get('gameId');
+    this.state.socket.on('room joined', () => {
+      const params = parseRoute(window.location.hash).params;
+      const gameId = params.get('gameId');
+      fetch(`/api/games/${gameId}`)
+        .then(res => res.json())
+        .then(result => {
+          this.setState({ meta: result });
+        });
+    });
+
+    const params = parseRoute(window.location.hash).params;
```

Comment:
Are you using hash routing yet? it'd be less redundant to access `this.context.route` instead of repeatedly parsing the hash.